README.md

```diff
@@ -4,7 +4,7 @@ This is a disassembly of Pokémon FireRed and LeafGreen.
 
 It builds the following ROM:
 
-* **pokefirered.gba** `sha1: 41cb23d8dccc8ebd7c649cd8fbb58eeace6e2fdc`
+* [**pokefirered.gba**](https://datomatic.no-intro.org/?page=show_record&s=23&n=1616) `sha1: 41cb23d8dccc8ebd7c649cd8fbb58eeace6e2fdc`
```

Comment:
What is this URL for? It's blocked by my firewall

Reply:
it's info about the dump

Reply:
It's for DAT-O-MATIC's info on the English FireRed v1.0 ROM dump.